docs/migration-guide.md

```diff
@@ -0,0 +1,330 @@
+# Migration Guide
+
+This document outlines how to migrate a repo using `@open-wc/testing` helpers and/or the first generation [@brightspace-ui/visual-diff](https://github.com/BrightspaceUI/visual-diff) library to `@brightspace-ui/testing`.
+
+Before starting, install `@brightspace-ui/testing` as a `devDependency` and include it in the repository's `package.json`. Similarly, once the migration is complete, `@open-wc/testing` can be removed.
+
+## Unit Tests
+
+Unit tests using the `@open-wc/testing` helpers can easily be migrated to use `@brightspace-ui/testing`'s versions with a few minor changes.
+
+### Step 1: Adjust Imports
+
+In test import statements, search and replace `from '@open-wc/testing'` with `from '@brightspace-ui/testing'`.
+
+Before:
+
+```javascript
+import { fixture, oneEvent } from '@open-wc/testing';
+```
+
+After:
+
+```javascript
+import { fixture, oneEvent } from '@brightspace-ui/testing';
+```
+
+### Step 2: Sinon Fake Timers
+
+If any tests are using [Sinon's useFakeTimers](https://sinonjs.org/releases/latest/fake-timers/), ensure that the scope of what gets faked is limited.
+
+For example, if a test requires that `setTimeout` be faked, pass `{ toFake: ['setTimeout'] }` to Sinon. This prevents Sinon from faking `requestAnimationFrame`, which will block calls to the new `fixture`.
+
+Before:
+
+```javascript
+import { useFakeTimers } from 'sinon';
+
+describe('my tests', () => {
+
+  let clock;
+  beforeEach(() => clock = useFakeTimers());
+  afterEach(() => clock.restore());
+
+});
+```
+
+After:
+
+```javascript
+import { useFakeTimers } from 'sinon';
+
+describe('my tests', () => {
+
+  let clock;
+  beforeEach(() => clock = useFakeTimers(
+	toFake: ['setTimeout']
+  }));
+  afterEach(() => clock.restore());
+
+});
+```
+
+### Step 3: No need to wait for `updateComplete`
+
+The `fixture()` in `@brightspace-ui/testing` differs from the `@open-wc/testing` version in that it waits for ALL nested Lit components' `updateComplete` to resolve.
+
+Therefore, any tests which were previously manually `await`-ing various elements' `updateComplete` no longer need to do so. 🎉
+
+Before:
+
+```javascript
+const elem = await fixture(html`
+  <some-elem>
+    <some-other-elem></some-other-elem>
+  </some-elem>
+`);
+await elem.shadowRoot.querySelector('nested-elem').updateComplete;
+await elem.querySelector('some-other-elem').updateComplete;
+```
+
+After:
+
+```javascript
+const elem = await fixture(html`
+  <some-elem>
+    <some-other-elem></some-other-elem>
+  </some-elem>
+`);
+```
+
+However, waiting for all nested `updateComplete`s means `fixture()` can be a bit slower. Therefore, tests that were relying on waiting for an event fired during the initial render will likely fail due to the event now firing during the `fixture()` call.
+
+Those tests will need to be reworked to not rely on events fired during `render()`.
+
+### Step 4: Use Browser Interaction Helpers (Optional)
```

Comment:
Minor: I know this technically _is_ optional, but I've found the new helpers so so so much better and I'm not sure this even needs to be called out as optional.  I'd highly recommend switching, everything I've touched is so much cleaner after switching to them.